Requesting a pull to mozilla:master from k88hudson:t1903

k88hudson opened the pull request.

mjschranz committed:
[#2199] Pressing delete while a trackevent is selected now closes the open editor and then reopens the plugin list

Commit:
[t1903] Fixed aspect ratio and overflow problems with image

mjschranz commented:
/botio check

ghost commented:
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/66ec74e85f5c990/output.txt

mjschranz commented:
/botio preview

ghost commented:
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/6c21f9bc16f7224/output.txt

ghost commented:
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/66ec74e85f5c990/output.txt

Total script time: 1.32 mins

ghost commented:
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/6c21f9bc16f7224/output.txt

Total script time: 0.70 mins

Server is running on http://pr972.butyr.org/templates/basic/
Run the Test Suite: http://pr972.butyr.org/test/

secretrobotron committed:
[t2249] more robust savedDataUrl loading mechanism

[t2249] review changes (async)

1. Reworked all of saved data loading on startup to be async.
2. Added XHR function, `getUntilComplete` to simplify external syntax for `XHR.get`.
3. Added extraRequestHeaders param to XHR.get.
4. Provided documentation for XHR functions and saved-data startup process.

[t2249] augmented test suite for query string load

[t2249] moved scrubber's destroy function

[t2249] explicit scrubber init

[t2249] fixed references to checkMedia

[t2249] media destroy back in place after rebase

[t2249] replaced timeline mediaupdate mechanism

[t2249] shrinking diff

mjschranz committed:
[#2322] Removing a layer now doesn't leave any trackevents still displaying

secretrobotron committed:
[t2325] patched drop handler in track-container

secretrobotron commented on templates/assets/plugins/image/popcorn.image.js:
```diff
@@ -77,12 +77,20 @@
-            ( _link || _container ).appendChild( _image );
+            _link.appendChild( _image );
+            _container.appendChild( _link );
+            if ( _image.getBoundingClientRect().height > _image.getBoundingClientRect().width  ) {
```
Cache the value of `getBoundingClientRect()`.

jbuck committed:
Bug 2331 - Remove unused cookie parser

secretrobotron commented:
/botio preview

ghost commented:
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @secretrobotron received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/f593309d1311ae3/output.txt

secretrobotron commented:
/botio check

ghost commented:
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @secretrobotron received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/076f2b2ffdc521c/output.txt

ghost commented:
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/f593309d1311ae3/output.txt

Total script time: 1.42 mins

Server is running on http://pr972.butyr.org/templates/basic/
Run the Test Suite: http://pr972.butyr.org/test/

ghost commented:
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/076f2b2ffdc521c/output.txt

Total script time: 1.47 mins

Commit:
[#2332] Fix up error tooltip creation and handling for project naming

[#2332] Add tooltip-error class to correct element

mjschranz commented:
/botio check

ghost commented:
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @mjschranz received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/aa799f0a57efa21/output.txt

mjschranz commented:
/botio preview

ghost commented:
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @mjschranz received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/cd6ffebf58e1acf/output.txt

ghost commented:
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/aa799f0a57efa21/output.txt

Total script time: 1.71 mins

ghost commented:
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/cd6ffebf58e1acf/output.txt

Total script time: 1.70 mins

Server is running on http://pr972.butyr.org/templates/basic/
Run the Test Suite: http://pr972.butyr.org/test/
